### PR TITLE
"View all" buttons to primary variant

### LINKED ISSUE #614: use a different button style for view all docs buttons
The view all docs button is very subtle, lets use a more prominent button style so this action is more visible. 

`variant={Button.VARIANT.PRIMARY}`

Instead of the subtle button use the style that is found on the integrations section of the home page. 

Screen Shot 2021-01-21 at 4 57 36 PM

### PR DESCRIPTION
**Pull request by caylahamann**
closes #614 

Changes the view all docs buttons for the TOC buttons to be of the variant primary and added it to the manual edits! Also ran a migration in this process.

### CODE REVIEW

File: src/content/docs/agents/python-agent/web-frameworks-servers/python-agent-fastapi-web-framework.mdx [hunk 1]
```diff
@@ -25,3 +25,4 @@ You can use the [Python agent API](/docs/agents/python-agent/python-agent-api) t
 ## Event loop diagnostic support [#event-loops]
 
 Our Python agent supports `asyncio` event loop diagnostics. For more information, see [Python event loop diagnostics](/docs/agents/python-agent/supported-features/python-event-loop-diagnostics).
+cs).
```
Comment (jerelmiller) [inline src/content/docs/agents/python-agent/web-frameworks-servers/python-agent-fastapi-web-framework.mdx:28]: Can we revert this change?

File: src/content/docs/agents/python-agent/web-frameworks-servers/python-agent-starlette-web-framework.mdx [hunk 2]
```diff
@@ -25,3 +25,4 @@ You can use the [Python agent API](/docs/agents/python-agent/python-agent-api) t
 ## Event loop diagnostic support [#event-loops]
 
 Our Python agent supports `asyncio` event loop diagnostics. For more information, see [Python event loop diagnostics](/docs/agents/python-agent/supported-features/python-event-loop-diagnostics).
+cs).
```
Comment (jerelmiller) [inline src/content/docs/agents/python-agent/web-frameworks-servers/python-agent-starlette-web-framework.mdx:28]: Can we revert this change?

File: src/content/docs/agents/python-agent/web-frameworks-servers/uvicorn.mdx [hunk 3]
```diff
@@ -26,3 +26,4 @@ You can use the [Python agent API](/docs/agents/python-agent/python-agent-api) t
 ## Event loop diagnostic support [#event-loops]
 
 Our Python agent supports `asyncio` event loop diagnostics. For more information, see [Python event loop diagnostics](/docs/agents/python-agent/supported-features/python-event-loop-diagnostics).
+cs).
```
Comment (jerelmiller) [inline src/content/docs/agents/python-agent/web-frameworks-servers/uvicorn.mdx:29]: Can we revert this change?

**Review by jerelmiller — CHANGES_REQUESTED** [review]
The changes look good, but looks like a few files got pulled in that we should revert. Could you also revert all the images? For some reason the changes to those images result in a corrupted image and cause build failures. If this PR could only include the changes to the `index.mdx` files, that would be great!

**Review by jerelmiller — COMMENTED** [review]
It looks like there are still some images that are part of the diff. Did the previous commit fail to build due to a corrupted image?

**Comment by caylahamann** [comment]
@jerelmiller now I think I finally got all of them, and yeah it failed because of a corrupted image

**Review by jerelmiller — APPROVED** [review]
Thanks for reverting those files!!
